### PR TITLE
Moved `zeppelin-solidity` to deps, moved `cross-conf-env` to devDeps

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
Since we want users to import `.sol` contracts and compile them from source, we need to have `zeppelin-solidity` as a dep and not devDep, otherwise users would not be able to compile.

Also `cross-conf-env` should be a devDep.

**Please republish npm package after merging this PR.**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,8 +63,7 @@
     "web3-typescript-typings": "^0.9.8",
     "web3-utils": "^1.0.0-beta.29",
     "when": "^3.7.8",
-    "yargs": "^11.0.0",
-    "zeppelin-solidity": "^1.6.0"
+    "yargs": "^11.0.0"
```
Comment (orenyodfat) [inline package.json]: do we need yargs ?

Comment (dev-matan-tsuberi) [reply package.json]: not anymore actually, since https://github.com/daostack/arc/pull/347

Comment (dev-matan-tsuberi) [reply package.json]: updated: https://github.com/daostack/arc/pull/357/commits/3acbe39c04b10f6f5699e96068865757c15aaea7

**Comment by dev-matan-tsuberi** [comment]
@orenyodfat Can you please republish the npm package?

**Pull request merged**